### DISCUSSION TITLE
stoicism/3/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#21] [General]
# stoicism/3/

https://shanenull.com/stoicism/3/

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2024/evening_ritual/

**Reply by shane0** [reply]
Hermitage
https://youtu.be/LQNHLO7s1N4?si=1Q7ydWRk3XnOXXUH

**Comment by shane0** [comment]
# 2024 update

- finish all chung tai courses this is going to take 6 more years including reading edward conze full 8000 10k & 250k line versions of the prajna paramita
- reread and compile comparisons of cormac books and movies including the chaconne bass lines
- complete maranasati preparation

**Reply by shane0** [reply]
shifting my goal I am delaying my goal of trying out nomad vanlife remote work still working remotely from home near the chan temple until I finish I think +6 more years of classes & loving it

**Comment by shane0** [comment]
# sustainable societies

* **Africa:**
    * Hadza people of Tanzania
    * San people of the Kalahari Desert
* **Asia:**
    * Sentinelese people of the Andaman Islands
    * Indigenous tribes in the Amazon rainforest
* **North America:**
    * Inuit people of the Arctic

**Comment by shane0** [comment]
# quotes

- quote from buddhism / movie goodrich
- in the end three things matter 
- did you love & live gently & let go gracefully
